README.md

```diff
@@ -23,13 +23,28 @@ be printed to your screen! Amazing, right?
 
 ## Use it!
 
+### As a base image:
+
 You can also use this image as a base image for your Carp-related Docker
 projects. This is as simple as starting your Dockerfile with:
 
 ```
 FROM carplang/carp:latest
 ```
 
+### As a carp builder:
+
+``` bash
+# from this repo root:
+docker run -v $(pwd)/example:/mnt/app \
+           --user 1000:1000 \
```

Comment:
this says to run container as a specified USER_ID:GROUP_ID  this way builder artifacts will be owned by the user.